src/standard/path.ml

```diff
@@ -37,3 +37,5 @@ let absolute path name = Absolute { path; name; }
 let rename f = function
   | Local { name; } -> local (f name)
   | Absolute { path; name; } -> absolute path (f name)
+
+let get_name (Local { name; } | Absolute { name; _ }) = name
```

Comment:
This is a dangerous function, and it very likely should not exist: it makes you lose the path information from absolute paths.

Reply:
Ok, I thought it would be a more straight-forward way to get the base-name of local/global variables in the cases in which the path is always empty (like in Alt-Ergo).

Reply:
Well, it's the case for problems coming from the alt-ergo native language (and most languages currently), but it will change with smtlibv3, where there will be modules and module paths used.

Reply:
Ok, I removed it.